### PR TITLE
75287 create saved filter table

### PR DESCRIPTION
**Pull request by llykkelarsen**

### CODE REVIEW

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Equinor.Procosys.Preservation.Domain.Audit;
+
+namespace Equinor.Procosys.Preservation.Domain.AggregateModels.PersonAggregate
+{
+    public class SavedFilter : PlantEntityBase, ICreationAuditable
+    {
+        public const int TitleLengthMax = 255;
+        public const int CriteriaLengthMax = 1024;
```
Comment (paal123) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:9]: Compare this to main. I suspect 2014 might be too short

Comment (paal123) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:9]: But we can ofcourse extend if we need later

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Equinor.Procosys.Preservation.Domain.Audit;
+
+namespace Equinor.Procosys.Preservation.Domain.AggregateModels.PersonAggregate
+{
+    public class SavedFilter : PlantEntityBase, ICreationAuditable
```
Comment (paal123) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:6]: This is missing link/FK to the person who owns it. Using createdyBy might be possible, but is seems a little strange maybe?

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        public const int TitleLengthMax = 255;
+        public const int CriteriaLengthMax = 8000;
+
+        public SavedFilter(string plant) : base(plant)
```
Comment (janid1967) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:11]: Remove plant as param and use base(null)

Comment (paal123) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:11]: But we still need plant? As the parameters in the filter might be plant-specific?
 @janid1967

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:11]: It's a constructor not to be used. Empty constructor needed for EF. See all other entities

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        {
+        }
+        public SavedFilter(string plant, string title, string criteria)
+            : base(plant)
+        {
+            Title = title;
+            Criteria = criteria;
+        }
+        public string Title { get; }
+        public string Criteria { get; }
```
Comment (janid1967) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:21]: How will criteria by saved? Json?

Comment (llykkelarsen) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/PersonAggregate/SavedFilter.cs:21]: Yes

**Pull request merged**